android/gradle.properties

```diff
@@ -8,4 +8,4 @@ org.gradle.jvmargs=-Xmx1024m -Xms256m
 android.useAndroidX=true
 android.native.buildOutput=verbose
 # Autogenerated by tools/unix/generate_localizations.sh
-supportedLocalizations=ar,be,bg,ca,cs,da,de,el,en,en_GB,es,es_MX,et,eu,fa,fi,fr,fr_CA,he,hi,hu,id,in,it,ja,ko,mr,nb,nl,pl,pt,pt_BR,ro,ru,sk,sv,sw,th,tr,uk,vi,zh,zh_HK,zh_MO,zh_TW
+supportedLocalizations=ar,be,bg,ca,cs,da,de,el,en,en_GB,es,es_MX,et,eu,fa,fi,fr,fr_CA,iw_IL,hi,hu,id,in,it,ja,ko,mr,nb,nl,pl,pt,pt_BR,ro,ru,sk,sv,sw,th,tr,uk,vi,zh,zh_HK,zh_MO,zh_TW
```

Comment:
Does using `iw` instead of `iw_IL` work?

Reply:
yes, it does. Updated to `iw`.